docs/products/kafka/kafka-mirrormaker/howto/exactly-once-delivery.md

```diff
@@ -0,0 +1,246 @@
+     --replication_flow_config '{ "exactly_once_delivery_enabled": true }'
+  ```
+
+- Disable exactly-once delivery on an existing replication flow:
+
+  ```bash
+  avn mirrormaker replication-flow update <service_name> \
+     --source-cluster <source_cluster> \
+     --target-cluster <target_cluster> \
+     --replication_flow_config '{ "exactly_once_delivery_enabled": false }'
+  ```
+
+Parameters:
+
+- `<service_name>`: Name of your Aiven for Apache Kafka MirrorMaker 2 service.
+- `<source_cluster>`: Alias of the source Aiven for Apache Kafka cluster.
+- `<target_cluster>`: Alias of the target Aiven for Apache Kafka cluster.
+- `exactly_once_delivery_enabled`: Set to `true` to enable or `false` to disable
+  exactly-once delivery.
+
+</TabItem>
+
+<TabItem value="api" label="Aiven API">
+
+Use the [ServiceKafkaMirrorMakerCreateReplicationFlow](https://api.aiven.io/doc/#tag/Service:_Kafka_MirrorMaker/operation/ServiceKafkaMirrorMakerCreateReplicationFlow) API to create or update a replication flow
+with `exactly_once_delivery_enabled` in the configuration.
+
+- Create a new replication flow with exactly-once delivery enabled:
+
+  ```bash
+  curl -X POST "https://console.aiven.io/v1/project/<project_name>/service/<service_name>/mirrormaker/replication-flows" \
+     -H "Authorization: Bearer <API_TOKEN>" \
+     -H "Content-Type: application/json" \
+     -d '{
+           "source_cluster": "<source_cluster>",
+           "target_cluster": "<target_cluster>",
+           "enabled": true,
+           "exactly_once_delivery_enabled": true
+         }'
+  ```
+
+- Update an existing replication flow to enable exactly-once delivery:
+
+  ```bash
+  curl -X PUT "https://console.aiven.io/v1/project/<project_name>/service/<service_name>/mirrormaker/replication-flows/<source_cluster>/<target_cluster>" \
+     -H "Authorization: Bearer <API_TOKEN>" \
+     -H "Content-Type: application/json" \
+     -d '{
+           "exactly_once_delivery_enabled": true
+         }'
+  ```
+
+- Update an existing replication flow to disable exactly-once delivery:
+
+  ```bash
+  curl -X PUT "https://console.aiven.io/v1/project/<project_name>/service/<service_name>/mirrormaker/replication-flows/<source_cluster>/<target_cluster>" \
+     -H "Authorization: Bearer <API_TOKEN>" \
+     -H "Content-Type: application/json" \
+     -d '{
+           "exactly_once_delivery_enabled": false
+         }'
+  ```
+
+Parameters:
+
+- `<project_name>`: Name of your project.
+- `<service_name>`: Name of your Aiven for Apache Kafka MirrorMaker 2 service.
+- `API_TOKEN`: Your personal Aiven API [token](/docs/platform/howto/create_authentication_token).
+- `<source_cluster>`: Alias of the source Aiven for Apache Kafka cluster.
+- `<target_cluster>`: Alias of the target Aiven for Apache Kafka cluster.
+- `exactly_once_delivery_enabled`:  Set to `true` to enable or `false` to disable
+  exactly-once delivery.
+
+</TabItem>
+<TabItem value="terraform" label="Terraform">
```

Comment:
Do we definitely want to include this sample in the Aiven Docs? We have information on the fields in our [Terraform documentation](https://registry.terraform.io/providers/aiven/aiven/latest/docs/resources/kafka_mirrormaker), so this would be duplicating info. 

This seems like a lot to document for one field. I wouldn't include things like a variables file here since if/how a user does this depends on their setup, internal guidelines, etc. We could instead add this field to the example usage in the TF docs if you think that would be helpful. What do you think?

Reply:
Thanks, @staceysalamon-aiven! I agree that the original example was too detailed for a single field, so I’ve simplified it. However, I believe it’s important to include Terraform steps here for the operation, as they provide immediate value to users. A link to the Terraform provider documentation is also included for those who need more detailed information.
cc: @ArthurFlag